src/config/enums.ts

```diff
@@ -0,0 +1,19 @@
+import { Values } from "../types/utils";
+
+export const HowKnown = {
```

Comment:
was this not anywhere else in the code?

Reply:
Just found it in the generated prisma client code. I'll update it